arbitrum-docs/stylus/concepts/stylus-gas.md

```diff
@@ -1,20 +1,29 @@
 ---
-title: 'Gas Costs in Stylus'
-sidebar_label: 'Gas Costs in Stylus'
-description: 'A conceptual overview of the pricing models of Stylus, including the relationshup between gas and ink.'
+title: 'Gas and ink (Stylus)'
+sidebar_label: 'Gas and ink'
```

Comment:
```suggestion
sidebar_label: 'Gas and ink in Stylus'
```
Kind of redundant, but since this doc is in a folder with something non-Stylus related, I think it's worth making explicitly clear this is Stylus related

Reply:
Everything in this folder is Stylus-specific, so I'd probably lean into the "keep "(Stylus)" qualifiers out of the sidebar" norm - I have faith that our readers will notice where these docs are in the sidebar ^^